spacy/gold.pyx

```diff
@@ -142,7 +142,7 @@ class GoldCorpus(object):
                 continue
             elif path.is_dir():
                 paths.extend(path.iterdir())
-            elif path.parts[-1].endswith(".json"):
+            elif path.parts[-1].endswith([".json", ".jsonl"]):
```

Comment:
```suggestion
            elif path.parts[-1].endswith((".json", ".jsonl")):
```

I have no idea if this matters but I think it typically takes a tuple instead of a list. So we might as well do it here for consistency.